examples/example.yaml

```diff
@@ -24,9 +24,16 @@ spec:
         - ReadWriteOnce
         resources:
           requests:
+          #   cpu: "4"
```

Comment:
again please remove all commented lines or fix the API values.  The resources are under the incorrect stanza.